h2/src/test/org/h2/test/db/TestCompatibilityOracle.java

```diff
@@ -284,6 +286,25 @@ private void testDate() throws SQLException {
         conn.close();
     }
 
+    private void testSequenceNextval() throws SQLException {
+        // Test NEXTVAL without Oracle MODE should return BIGINT
+        checkSequenceTypeWithMode("REGULAR", Types.BIGINT);
+        // Test NEXTVAL with Oracle MODE should return DECIMAL
+        checkSequenceTypeWithMode("Oracle", Types.DECIMAL);
```

Comment:
The Oracle's JDBC driver returns `Types.NUMERIC`, isn't it?

As I already said, H2 returns either `Types.NUMERIC` or `Types.DECIMAL` depending on settings. I think that
```Java
SysProperties.BIG_DECIMAL_IS_DECIMAL ? Types.DECIMAL : Types.NUMERIC
```
should be used here to avoid failure when test suit is launched with non-default settings.